proxy/.env

```diff
@@ -0,0 +1 @@
+REDIS="rediss://default:81c887ac3bdb4d0ab49b31851832c57a@apn1-electric-asp-34359.upstash.io:34359"
```

Comment:
Is it okay to expose this?

Reply:
No, I will also need to change the API key now